StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet.swift

```diff
@@ -104,6 +104,9 @@ public class PaymentSheet {
     }
 
     required init(mode: InitializationMode, configuration: Configuration) {
+        assert(configuration.returnURL != nil,
```

Comment:
It crossed my mind this might be too aggressive, and maybe we should assert the return URL in PaymentSheetLoader or something, but I figure the sooner in the process to assert the better.